be/src/vec/functions/llm/functions_llm.h

```diff
@@ -51,7 +58,14 @@ class LLMFunction : public IFunction {
 
     Status execute_impl(FunctionContext* context, Block& block, const ColumnNumbers& arguments,
                         uint32_t result, size_t input_rows_count) const override {
-        MutableColumnPtr col_result = ReturnType::create();
+        MutableColumnPtr col_result;
```

Comment:
why here call `get_return_type_impl()::create()` ? And Remove the template param `ReturnType`?